packages/manager/apps/pci-databases-analytics/src/pages/services/[serviceId]/indexPatterns/IndexPatterns.page.tsx

```diff
@@ -0,0 +1,107 @@
+import { useTranslation } from 'react-i18next';
```

Comment:
Same as `DeleteIndexModal` component, with `strictNullChecks` on, `service` & `projectId` are possibly undefined and can lead to runtime errors